web/rootfs/defaults/meet.conf

```diff
@@ -50,6 +52,22 @@ location = /http-bind {
     proxy_set_header Host {{ .Env.XMPP_DOMAIN }};
 }
 
+{{ if $ENABLE_XMPP_WEBSOCKET }}
+# xmpp websockets
+location = /xmpp-websocket {
+    proxy_pass {{ .Env.XMPP_BOSH_URL_BASE }}/xmpp-websocket;
+    proxy_http_version 1.1;
+
+    proxy_set_header Connection "upgrade";
+    proxy_set_header Upgrade $http_upgrade;
+
+    proxy_set_header Host {{ .Env.XMPP_DOMAIN }};
+    proxy_set_header X-Forwarded-For $remote_addr;
+    proxy_read_timeout 60s;
```

Comment:
Can we drop this timeout?

Reply:
We could drop the timeout, but the nginx default is still `proxy_read_timeout 60s;`

Do you want to drop the line or do you want to remove the timeout?

Reply:
if you are talking about mod_websocket_smacks.patch, you can skip it. We recently discussed with Matt the patches and that one is not needed and we are currently running without it on meet.jit.si ... maybe we can also remove it from jitsi-meet repo.

Reply:
> Do you want to drop the line or do you want to remove the timeout?

The line please.

Reply:
Thanks @damencho !